README.md

```diff
@@ -547,6 +561,20 @@ UTEST(foo, bar) {
 }
 ```
 
+### ASSERT_NEAR(x, y, epsilon)
```

Comment:
Typo here? I assume it should be `EXPECT_NEAR`

Reply:
Good catch! Fixed.